components/brave_wallet_ui/utils/format-prices.ts

```diff
@@ -0,0 +1,92 @@
+import BigNumber from 'bignumber.js'
```

Comment:
We're missing copyright headers in quite a few files in this directory. Can you make sure all of the files being added/modified in this PR are making sure to include this Header?